res/css/views/messages/_ViewSourceEvent.scss

```diff
@@ -19,8 +19,10 @@ limitations under the License.
     opacity: 0.6;
     font-size: $font-12px;
     width: 100%;
+    overflow-x: auto; // Cancel overflow setting of .mx_EventTile_content
 
-    pre, code {
+    pre,
+    code {
```

Comment:
This doesn't seem to be necessary?

Reply:
I remember I've read somewhere that selectors should be separated like that, while it is fine to revert it.

Reply:
You might be right - let me check

Reply:
I could drop the latest commit :-D

Reply:
You were right, sorry for the confusion in both comments... :sweat_smile:  Could you please revert the revertion? :D

Reply:
Done!

Reply:
Thank you!